config.go

```diff
@@ -122,6 +124,53 @@ func (h hypervisor) image() string {
 	return h.Image
 }
 
+func (h hypervisor) kernelParams() string {
+	if h.KernelParams == "" {
+		return defaultKernelParams
+	}
+
+	return h.KernelParams
+}
+
+func deserializeParams(paramString string) []vc.Param {
```

Comment:
Please create this function in virtcontainers as it can be reused by any consumer of the virtcontainers library.